Airwallex/Card/Internal/AWXWebViewController.m

```diff
@@ -47,58 +50,76 @@ - (void)viewDidLoad
     webView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
     [self.view addSubview:webView];
     self.webView = webView;
-    
-    self.navigationItem.leftBarButtonItem = [[UIBarButtonItem alloc] initWithImage:[[UIImage imageNamed:@"close" inBundle:[NSBundle resourceBundle]] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate] style:UIBarButtonItemStylePlain target:self action:@selector(cancel:)];
-    
+
+    self.navigationItem.leftBarButtonItem =
+        [[UIBarButtonItem alloc] initWithImage:[[UIImage imageNamed:@"close" inBundle:[NSBundle resourceBundle]]
+                                                   imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate]
+                                         style:UIBarButtonItemStylePlain
+                                        target:self
+                                        action:@selector(cancel:)];
+
     [self.webView loadRequest:self.urlRequest];
 }
 
-- (void)cancel:(id)sender
-{
-    [self dismissViewControllerAnimated:YES completion:^{
-        if (self.webHandler) {
-            self.webHandler(nil, [NSError errorWithDomain:AWXSDKErrorDomain code:-1 userInfo:@{NSLocalizedDescriptionKey: NSLocalizedString(@"User cancelled.", nil)}]);
-        }
-    }];
+- (void)cancel:(id)sender {
+    [self dismissViewControllerAnimated:YES
+                             completion:^{
+                                 if (self.webHandler) {
+                                     self.webHandler(nil, [NSError errorWithDomain:AWXSDKErrorDomain
+                                                                              code:-1
+                                                                          userInfo:@{
+                                                                              NSLocalizedDescriptionKey:
+                                                                                  NSLocalizedString(@"User "
+                                                                                                    @"cancelled.",
+                                                                                                    nil)
+                                                                          }]);
```

Comment:
I feel like blocks are the hardest to be formatted correctly

Reply:
it's a bit annoying - Objective-C always wants to align to colon placement